Client/Components/SingleAnswerComponent.razor

```diff
@@ -1,133 +1,131 @@
+        <div class="row-line">
+        </div>
+        <div class="row-unloggedUser">
+            <UnloggedUserComponent></UnloggedUserComponent>
+        </div>
     }
-    else
-    {
-        <div class="answer-text">
-            @if (Answer.IdAnswer == ActiveResponse)
+    <div class="row-answer">
+        <div class="text @(ActiveUser==null?"blur":"")">
+            @Answer.Text
+        </div>
+
+        <div class="footer @(ActiveUser==null?"blur":"")">
+            <button class="quanda-icon icon-arrowup answer-rating-arrow" style="@(Answer.IsLiked==1?"color:#FFAD33":"")" @onclick="@(async() => await IncreaseRating())" disabled="@(isNonInteractive || OwnerOfAnswer || Answer.IsLiked == 1)" />
+            <div class="answer-rating-text">
+                @Answer.Rating
+            </div>
+            <button class="quanda-icon icon-arrowdown answer-rating-arrow" style="@(Answer.IsLiked==-1?"color:#FFAD33":"")" @onclick="@(async() => await DecreaseRating())" disabled="@(isNonInteractive || OwnerOfAnswer || Answer.IsLiked == -1)" />
+            @if (Answer.IdRootAnswer == null)
             {
-                <div style="background-color:lightgoldenrodyellow; color: black">
-                    @Answer.Text
+                <div class="black-action" @onclick="@(async () =>await SetReplayActive())">
+                    replay
                 </div>
             }
-            else
+            @if (OwnerOfAnswer)
             {
-                <div >
-                    @Answer.Text
+                <div class="black-action" @onclick="@(async () =>await EditAnswer())">
+                    <button class="quanda-icon icon-edit" />
+                    <div>
+                        edit
+                    </div>
+                </div>
+                <div class="red-action" @onclick="@(()=> DeleteAnswer.InvokeAsync(Answer))">
+                    <button class="quanda-icon icon-delete red" />
+                    <div>
+                        delete
+                    </div>
                 </div>
             }
-        </div>
-
-
-        <div class="edit-option-text">
-            @if (OwnerOfAnswer)
+            else
             {
-                <div>
-                    <button @onclick="@SetEditMode">E</button>
-                </div>
-                <div>
-                    <button @onclick="@(async () =>await DeleteAnswer())">X</button>
+                <div class="red-action" @onclick="@(async () =>await ReportAnswer())">
+                    <button class="quanda-icon icon-report red" />
+                    <div>
+                        Report answer
+                    </div>
                 </div>
-
             }
-            <div>
-                <button @onclick="@(() => OnSetReplyAnswer.InvokeAsync(Answer.IdAnswer))">R</button>
-            </div>
         </div>
-    }
 
+        @if (IsReplayActive)
+        {
+            <div style="margin-left: 80px;">
```

Comment:
Please use css classes instead of styles as it will be easier to change them later.